lib/ansible/modules/systemd.py

```diff
@@ -551,6 +600,9 @@ def main():
                 # this should not happen?
                 module.fail_json(msg="Service is in unknown state", status=result['status'])
 
+    if lingered:
+        disable_linger(module)
```

Comment:
Disabling linger here will most likely result in the user unit not working as intended. In order for user units to start on boot and/or not be killed when a login session for the user account no longer exists, the user account must have linger permanently enabled.